app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,25 @@
+@charset "UTF-8";
```

Comment:
how come we need this explicitly?

Reply:
I've no idea. I think this line was already in this file when I renamed it. I'll try taking it out.

Reply:
Have removed. Seems to be okay.